sunpy/visualization/animator/base.py

```diff
@@ -526,7 +526,7 @@ def _sanitize_axis_ranges(self, axis_ranges, data_shape):
                     # and APIs using both [min, max] pair and manual definition of each pixel
                     # values can be unambiguously and simultanously supported.
                     extent += [axis_ranges[i][0], axis_ranges[i][-1]]
-                    axis_ranges[i] = edges_to_centers_nd(axis_ranges[i])
+                    axis_ranges[i] = edges_to_centers_nd(axis_ranges[i], i)
```

Comment:
Can you also see if this happens in other places?

Reply:
Sure, though it'll take some time.